commit 6e9787fcc677b15cb9fc0baea8de545127b4fb85
Author: Freddy Spierenburg <freddy@snarl.nl>
Date:   Mon Mar 25 14:35:17 2019 +0100

    DOC: The option httplog is no longer valid in a backend.
    
    This can be backported to 1.9 and 1.8.
    
    (cherry picked from commit e88b77351b3fe34a83d5208773a63f78fb140722)
    Signed-off-by: Willy Tarreau <w@1wt.eu>
    (cherry picked from commit 79b70b0f8f86315230323e7f826796dffba4b85b)
    Signed-off-by: Christopher Faulet <cfaulet@haproxy.com>

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 8d75d568..3608a897 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -2118,7 +2118,7 @@ option http-tunnel                   (*)  X          X         X         X
 option http-use-proxy-header         (*)  X          X         X         -
 option httpchk                            X          -         X         X
 option httpclose                     (*)  X          X         X         X
-option httplog                            X          X         X         X
+option httplog                            X          X         X         -
 option http_proxy                    (*)  X          X         X         X
 option independent-streams           (*)  X          X         X         X
 option ldap-check                         X          -         X         X
